R/regression.R

```diff
@@ -667,12 +667,14 @@ Regression <- function(formula = as.formula(NULL),
         }
         if (output == "Jaccard Coefficient")
```

Comment:
How about:
```
fun <- switch(output, "Jaccard Coefficient" = computeJaccardCoefficientOutput, computeCorrelationOutput)
result$importance <- fun(input.formula, .estimation.data, .weights, labels, correction)
result$importance <- output
```

Reply:
Addressed this and more streamlined improvements in https://github.com/Displayr/flipRegression/pull/24